### PR TITLE
{numlib}[system/system] cuDNN v8.9.2.26 w/ CUDA 12.2.0

### PR DESCRIPTION
**Pull request by surak**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by surak** [comment]
The older ones have multiple checksums for the different platforms. How to add that automatically?

File: easybuild/easyconfigs/c/cuDNN/cuDNN-8.9.2.26-CUDA-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# note: cuDNN is tied to specific to CUDA versions,
+# see also https://docs.nvidia.com/deeplearning/cudnn/support-matrix/index.html#cudnn-cuda-hardware-versions
+local_short_ver = '.'.join(version.split('.')[:3])
+local_cuda_major = '12'
+
+source_urls = [
+    'https://developer.download.nvidia.com/compute/cudnn/redist/cudnn/linux-%(cudnnarch)s/'
+]
+sources = ['%%(namelower)s-linux-%%(cudnnarch)s-%%(version)s_cuda%s-archive.tar.xz' % local_cuda_major]
+checksums = ['ccafd7d15c2bf26187d52d79d9ccf95104f4199980f5075a7c1ee3347948ce32']
```
Comment (branfosj) [inline easybuild/easyconfigs/c/cuDNN/cuDNN-8.9.2.26-CUDA-12.2.0.eb:19]:
```suggestion
checksums = [{
    '%%(namelower)s-linux-ppc64le-%%(version)s_cuda%s-archive.tar.xz' % local_cuda_major:
        '4f5e5bd01570c4805b93fb199f8bb6f8475d016948c55abf48fed9ffe89d13e5',
    '%%(namelower)s-linux-sbsa-%%(version)s_cuda%s-archive.tar.xz' % local_cuda_major:
        '898d00c82f9ad8797bd6f6c639327b320a38fa4aeebfb2b3fbb2db0d38f7e1b0',
    '%%(namelower)s-linux-x86_64-%%(version)s_cuda%s-archive.tar.xz' % local_cuda_major:
        'ccafd7d15c2bf26187d52d79d9ccf95104f4199980f5075a7c1ee3347948ce32',
}]
```

That is from https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/c/cuDNN/cuDNN-8.9.2.26-CUDA-12.1.1.eb - cuDNN is tied to the CUDA major version and the `x86_64` one is the same, so the other ones should be the same as well.

At the moment there is no automated way to generate these. it is a case of manually downloading them, running `sha256sum`, and updating the information in that block in the easyconfig.

**Review by branfosj — CHANGES_REQUESTED** [review]

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/125c0ec9356248615e4e7b46c230042b for a full test report.

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @surak!

**Pull request merged**